runtime/jcl/common/com_ibm_jvm_Stats.c

```diff
@@ -78,11 +80,16 @@ Java_com_ibm_jvm_Stats_getStats(JNIEnv *env, jobject obj)
 		softmx = max;
 	}
 
-	/* Calculation of free physical memory from existing function in mgmtosext.c*/
-	free = Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getFreePhysicalMemorySizeImpl(env, obj);
 
-	/* Calculation of total physical memory from port lib api */
-	tot = (jlong) j9sysinfo_get_physical_memory();
+	rc = j9sysinfo_get_memory_info(&memInfo);
+	
+	if (0 == rc) { 
+		/* Calculation of free physical memory from existing function in mgmtosext.c*/
```

Comment:
This comment is not more valid. Please remove it.

Reply:
Removed